docs/index.md

```diff
@@ -19,9 +19,19 @@ Then add `flags` as an installed app in your Django `settings.py`:
 
 ```python
 INSTALLED_APPS = (
-    ...
+    # …
     'flags',
-    ...
+    # …
+)
+```
+
+And `django.template.context_processors.request` to the `TEMPLATE_CONTEXT_PROCESSORS` setting:
+
+```python
+TEMPLATE_CONTEXT_PROCESSORS = (
```

Comment:
`TEMPLATE_CONTEXT_PROCESSORS` is deprecated.

Shouldn't we use the `TEMPLATES` settings style?

Reply:
🙈